internal/provider/data_source_oauth_client_test.go

```diff
@@ -137,35 +137,7 @@ func TestAccTFEOAuthClientDataSource_missingParameters(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config:      testAccTFEOAuthClientDataSourceConfig_missingParameters(rInt),
-				ExpectError: regexp.MustCompile("one of `name,oauth_client_id,service_provider` must"),
-			},
-		},
-	})
-}
-
-func TestAccTFEOAuthClientDataSource_missingOrgWithName(t *testing.T) {
-	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
-	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { testAccTFEOAuthClientDataSourcePreCheck(t) },
-		ProtoV5ProviderFactories: testAccMuxedProviders,
-		Steps: []resource.TestStep{
-			{
-				Config:      testAccTFEOAuthClientDataSourceConfig_missingOrgWithName(rInt),
-				ExpectError: regexp.MustCompile("all of `name,organization` must"),
-			},
-		},
-	})
-}
-
-func TestAccTFEOAuthClientDataSource_missingOrgWithServiceProvider(t *testing.T) {
-	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
-	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { testAccTFEOAuthClientDataSourcePreCheck(t) },
-		ProtoV5ProviderFactories: testAccMuxedProviders,
-		Steps: []resource.TestStep{
-			{
-				Config:      testAccTFEOAuthClientDataSourceConfig_missingOrgWithServiceProvider(rInt),
-				ExpectError: regexp.MustCompile("all of `organization,service_provider` must be"),
+				ExpectError: regexp.MustCompile("Invalid Attribute Combination"),
```

Comment:
I removed these tests since we are now defaulting organization to the provider config.